repo/mk/core.check.eclint.mk

```diff
@@ -17,8 +20,11 @@
 
 SF_IS_TRANSCRYPTED ?= false
 
+ECCHECKER = $(call which,ECCHECKER,editorconfig-checker)
 ECLINT = $(call npm-which,ECLINT,eclint)
-$(foreach VAR,ECLINT,$(call make-lazy,$(VAR)))
+$(foreach VAR,ECCHECKER ECLINT,$(call make-lazy,$(VAR)))
```

Comment:
`foreach` is a function in support_firecloud? What's wrong with `for x in ...`?
What is `make-lazy` and why didn't I find any definition of it when grepping for it in support_firecloud?

Reply:
This is a makefile. foreach is a make call

make-lazy def: https://github.com/andreineculau/core.inc.mk/blob/master/core.inc.mk
Pretty sure that grepping from the terminal (not github ui) finds it ;)